CodeEditModules/Modules/AppPreferences/src/Media.xcassets/Settings Image - Dark.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Don't use images for this. Make a custom SwiftUI View instead.

Reply:
There is no need to use SwiftUI for this image, it's just a static image like all other apps

Reply:
We will not include 9 images which are 670 by 440 pixels in size. Note that the scale factor in asset catalogs has a reason. Do not put the same image in 3 times. We want a small/lightweight app experience and including assets when it can be done in code is bad practice.

For reference this is completely generated and every color aspect can be customized since it is reusable. Your View would be a lot simpler at less than 100 lines of code.
The size of this Swift file is 8KB which will be even less once compiled. One of your images has already over 100KB. With 9 images we are at almost 1 MB. Way too much for simple thumbnails.

<img width="539" alt="Screen Shot 2022-03-30 at 14 42 23" src="https://user-images.githubusercontent.com/9460130/160836808-7114a835-a06e-4d34-80aa-6573f91ddd38.png">

Reply:
<img width="441" alt="CleanShot 2022-03-30 at 20 59 20@2x" src="https://user-images.githubusercontent.com/46642225/160839930-6f827440-a510-4739-859a-f92122b51d97.png">

You may have read the wrong content

Reply:
> We will not include 9 images which are 670 by 440 pixels in size. Note that the scale factor in asset catalogs has a reason. Do not put the same image in 3 times. We want a small/lightweight app experience and including assets when it can be done in code is bad practice.
> 
> For reference this is completely generated and every color aspect can be customized since it is reusable. Your View would be a lot simpler at less than 100 lines of code. The size of this Swift file is 8KB which will be even less once compiled. One of your images has already over 100KB. With 9 images we are at almost 1 MB. Way too much for simple thumbnails.
> 
> <img alt="Screen Shot 2022-03-30 at 14 42 23" width="539" src="https://user-images.githubusercontent.com/9460130/160836808-7114a835-a06e-4d34-80aa-6573f91ddd38.png">

I'm still working on this one, but won't be committing in this PR

Reply:
Now it only 25kb per image. And suitable for all the computer

Reply:
> > You may have read the wrong content
> 
> If you look at the files they are all the same size. I looked at each image file in your PR and they are all the same size.

fixed